spec/request/service_instances_spec.rb

```diff
@@ -2573,33 +2572,68 @@ def check_filtered_instances(*instances)
         }
       }
 
-      context 'and the update contains metadata only' do
-        it 'updates the metadata' do
-          api_call.call(admin_headers)
-          expect(last_response).to have_status_code(200)
-          expect(parsed_response.dig('metadata', 'labels')).to eq({ 'unit' => 'metre', 'distance' => '1003' })
-          expect(parsed_response.dig('metadata', 'annotations')).to eq({ 'location' => 'london' })
+      context 'and it is a create operation' do
```

Comment:
It's good that you added tests for the different operations being "in progress". As written in my other comment, it should be possible to update metadata during "create in progress"; so you could test this as well.